src/frontend/js/pages/TeacherCoursesDashboardLoader/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { Spinner } from 'components/Spinner';
+import { DashboardLayout } from 'widgets/Dashboard/components/DashboardLayout';
+import { TeacherProfileDashboardSidebar } from 'widgets/Dashboard/components/TeacherProfileDashboardSidebar';
+
+const messages = defineMessages({
+  loading: {
+    defaultMessage: 'Loading courses ...',
+    description: 'Message displayed while loading courses',
+    id: 'components.TeacherCoursesDashboardLoader.loading',
+  },
+  courseRuns: {
+    defaultMessage: 'Your courses',
+    description: 'Filtered courses title',
+    id: 'components.TeacherCoursesDashboardLoader.title.filteredCourses',
+  },
+  incoming: {
+    defaultMessage: 'Incomming',
+    description: 'Incoming courses title',
+    id: 'components.TeacherCoursesDashboardLoader.title.incoming',
+  },
+  ongoing: {
+    defaultMessage: 'Ongoing',
+    description: 'Ongoing courses title',
+    id: 'components.TeacherCoursesDashboardLoader.title.ongoing',
+  },
+  archived: {
+    defaultMessage: 'archived',
```

Comment:
```suggestion
    defaultMessage: 'Archived',
```

Reply:
I removed all of them, it'll add and use in the [next PR](https://github.com/openfun/richie/pull/1951)